### PR TITLE
luarocks: release new version 1.02

### PR DESCRIPTION
**Pull request by membphis**

### CODE REVIEW

File: rockspec/resty-redis-cluster-1.02-4.rockspec [hunk 1]
```diff
@@ -1,21 +1,18 @@
...
     summary = "Openresty lua client for redis cluster",
     detailed = [[
         Openresty environment lua client with redis cluster support.
-        This is a wrapper around the 'resty.redis' library with cluster discovery 
+        This is a wrapper around the 'resty.redis' library with cluster discovery
         and failover recovery support.
     ]],
     license = "Apache License 2.0"
 }
-dependencies = {
```
Comment (membphis) [inline rockspec/resty-redis-cluster-1.02-4.rockspec:15]: I remove this field `dependencies`, it is wrong if the user ENV is openresty.
we can write a doc if the user uses other ENV.

**Comment by membphis** [comment]
@steve0511 you can create a luarocks account and upload file `rockspec/resty-redis-cluster-1.02-4.rockspec`, then the use can install by command: `luarocks install resty-redis-cluster` ^_^

**Comment by steve0511** [comment]
Thanks! 

It's done. 
https://luarocks.org/modules/steve0511/resty-redis-cluster

**Comment by membphis** [comment]
congratulation ^_^

make a test right now, it works fine.

**Pull request merged**